src/JustEat.StatsD/TagsFormatters/NoOpTagsFormatter.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+
+namespace JustEat.StatsD.TagsFormatters
+{
+    internal sealed class NoOpTagsFormatter : IStatsDTagsFormatter
+    {
+        public bool AreTrailing { get; }
```

Comment:
I think this makes it more explicit in intent.

```suggestion
        public bool AreTrailing => false;
```

Reply:
Done!